packages/playground/vite.config.smallNodeApp.js

```diff
@@ -0,0 +1,31 @@
+// @ts-check
+import { sentryVitePlugin } from "@sentry/unplugin";
+import { defineConfig } from "vite";
+import * as path from "path";
+import placeHolderOptions from "./config.json";
```

Comment:
do we want to use the placeholderoptions in this file?

Reply:
Nope :D